doc/radosgw/multisite.rst

```diff
@@ -230,9 +230,9 @@ the zonegroup
     }
 
 
-  Note that the above ``--rgw-zonegroup`` and ``--default`` switches add the
-  zone to a zonegroup and makes it the default zone as well. This can also be
-  accomplished alternatively by the following commands::
+Note that the above ``--rgw-zonegroup`` and ``--default`` switches add the
+zone to a zonegroup and makes it the default zone as well. This can also be
+accomplished alternatively by the following commands::
 
     # radosgw-admin zone default --rgw-zone=us-east
     # radosgw-admin zonegroup add --rgw-zonegroup=us --rgw-zone=us-east
```

Comment:
@oritwas  We didn't create zone named `us-east` . All `us-east` should be  `us-east-1`. I'll cleanup those and update commit later.